src/Packetery/Module/WpdbAdapter.php

```diff
@@ -325,7 +325,8 @@ public function getLastWpdbError(): string {
 	 * @return \Generator
 	 */
 	public function getWpdbQueries(): \Generator {
-		if ( isset( $this->wpdb->queries ) ) {
+		if ( ( defined( 'SAVEQUERIES' ) && constant( 'SAVEQUERIES' ) === true )
```

Comment:
_Expressions in parentheses MAY be split across multiple lines, where each subsequent line is indented at least once. When doing so, the first condition MUST be on the next line. The closing parenthesis and opening brace MUST be placed together on their own line with one space between them. Boolean operators between conditions MUST always be at the beginning or at the end of the line, not a mix of both._ My používáme operátory na konci řádek.